examples/blockscaled_gemm_sm100/grouped_gemm_mxfp8_blockscaled_1d1d.py

```
@@ -0,0 +1,670 @@
+
+    half_N = block_N // 2
+    k_iters = T.ceildiv(K, block_K)
+    sf_load_period = sf_granularity_k * 4 // block_K
+    sf_k_groups = T.ceildiv(T.ceildiv(K, sf_granularity_k), 4)
+    assert sf_load_period == 4
+
+    A: T.Tensor[[M_total, K], in_dtype]
+    B: T.Tensor[[E, N, K] if transpose_B else [E, K, N], in_dtype]
+    SFA: T.Tensor[[sf_k_groups * M_total], T.uint32]
+    SFB: T.Tensor[[sf_k_groups * E * N], T.uint32]
+    offsets: T.Tensor[[E1], "int32"]
```

Ruff (0.15.11)

[error] Undefined name `int32`

(F821)